src/development/LocalTest/appsettings.json

```diff
@@ -36,8 +36,8 @@
   "AllowedHosts": "*",
   "LocalPlatformSettings": {
     "LocalTestingStorageBasePath": "c:/AltinnPlatformLocal/",
-    "AppRepsitoryBasePath": "C:\\Repos\\altinn-apps\\nina-testapp\\App\\",
-    "LocalTestingStaticTestDataPath": "C:/Repos/altinn-studio/src/development/TestData/"
+    "AppRepsitoryBasePath": "C:/Users/sekeberg/git/steffens-2020-dato-app/App/",
+    "LocalTestingStaticTestDataPath": "C:/Users/sekeberg/git/altinn-studio/src/development/TestData/"
```

Comment:
Maybe we should set some default value here, so we don't keep updating with our local paths :)
F.ex.
```
    "AppRepsitoryBasePath": "<Path to your app>/App/",
    "LocalTestingStaticTestDataPath": "<Path to altinn-studio repo>/altinn-studio/src/development/TestData/"
```
seems to work ok.

Reply:
Agreed, Ill fix :)